package.json

```diff
@@ -59,6 +60,7 @@
     "@typescript-eslint/parser": "^3.0.0",
     "babel-loader": "^8.0.4",
     "bower": "^1.8.8",
+    "check-engines": "^1.5.0",
```

Comment:
**question:** where do we need this dependency `check-engines`? I noticed that in the preinstall script we are using `check-engine` with npx

Reply:
Installing the dependency enforces the version of `check-engines` we use with npx. In addition, by default, running `npx check-engines` in the CI without the depencency defined would also fail as npm would prompt asking whether it is okay to install `check-engines` before running (and the CI wouldn't be able to respond either way without user input or a `--yes` flag included)

Reply:
that sounds sensible, but when do we use `check-engines` with npx? I can only see us using `check-engine` and not `check-engines`

Reply:
oof – that's a good point. I'll fix that for these PRs but that was a typo in my migration script so I'll need to run a nori script to fix it in other places at some point... I guess it's running `npx` even if the package isn't installed then? I don't know how that works!!

Reply:
yeah, if the package isn't installed then the latest version of `check-engine` will be installed upon running `npx check-engine`